ats/test/HelloATS/ats_check_log.py

```diff
@@ -1,26 +1,25 @@
-#!/usr/apps/ats/7.0.1/bin/python
+#!/usr/bin/env python3
 import os
 import sys
 
 if __name__ == '__main__':
-    if len(sys.argv) < 2:
+    if len(sys.argv) != 2:
         sys.exit(1)
```

Comment:
I know it's out of the scope of this PR but printing usage info could be helpful, maybe something like:
```
sys.exit("expected exactly 1 argument")
```
or
```
sys.exit("usage: {os.path.baseline(__file__)} ats.log")
```

Reply:
I totally agree and I'll keep this in mind for future updates. In general I'm trying to keep updates to a minimum but something like this is a safe and useful improvement. 👍